packages/atlas/src/components/NftTile/NftTile.tsx

```diff
@@ -12,85 +12,65 @@ import { Container } from './NftTile.styles'
 export type Member = {
   assetUrl?: string | null
   name: string
+  onClick?: () => void
+  loading?: boolean
 }
 
 export type NftTileProps = {
-  auction: 'none' | 'minBid' | 'topBid' | 'waiting'
-  buyNow?: boolean
-  thumbnail: VideoThumbnailProps
+  auction?: 'none' | 'minBid' | 'topBid' | 'waiting'
+  thumbnail?: VideoThumbnailProps
+  title?: string | null
+  owner?: Member
+  creator?: Member
   loading?: boolean
-  title: string
-  owner: Member
-  creator: Member
-  timer?: string
   duration?: number | null
   views?: number | null
-  bid: number
-  minBid?: number
-  topBid?: number
-  timeLeft?: number
+  buyNowPrice?: number | null
+  minBid?: number | null
+  topBid?: number | null
+  timeleft?: number
```

Comment:
Why did you change it from camel case?

Reply:
oh - I don't know I just thought that `timeleft` is an existing word 😆 . I'll fix this in a sec. 

Regarding counting - yes this was intentional, I move counting logic to NftTileViewer, because the `auction` prop relies on time left. You could see this here https://github.com/Joystream/atlas/pull/2189/files#diff-88aede35f9de0cbd66bbf341f5187f3c6c21bcea7abd030ab61299132e809cf7R100